3main

```diff
@@ -413,63 +405,26 @@ sleep 2
 if [[ $MINER_TEMP_CONTROL == YES ]]
 then
   if ps ax | grep -q '[6]tempcontrol'; then
-    if [[ $WTM_AUTO_SWITCH == NO ]]; then
-      ps -ef | awk '$NF~"6tempcontrol" {print $2}' | xargs kill
-    fi
+    ps -ef | awk '$NF~"6tempcontrol" {print $2}' | xargs kill
   fi
   sleep 2
   echo ""
   if ! ps ax | grep -q '[6]tempcontrol'; then
     echo "LAUNCHING:  MINER TEMPCONTROL"
     screen -c ${NVOC}/screenrc-tempcontrol -dmSL temp bash ${NVOC}/6tempcontrol
+    echo ""
     if [[ $LOCALorREMOTE == LOCAL ]]; then
-      echo ""
       echo "process in screen temp, guake terminal Tab (f12), 'nvOC temp-log'"
       #if guake tab is already showing temp control output dont open a new one
-      if [[ -z $(ps ax | grep "tail -f ${NVOC}/nvoc_logs/tempcontrol-screenlog.0" |grep -v grep) ]];then
+      #if [[ -z $(ps ax | grep "tail -f ${NVOC}/nvoc_logs/tempcontrol-screenlog.0" |grep -v grep) ]];then
+      if ! ps ax | grep "tail -f ${NVOC}/nvoc_logs/[t]empcontrol-screenlog.0"; then
```

Comment:
We can add a "-n 0" arg to tail so it wont show last lines

Reply:
which lines? those from previous executions?

Reply:
Yes, when tail starts without "-n 0 "it shows 10 lines of last execution
Do we need those?